controllers/user.js

```diff
@@ -787,6 +788,7 @@ async function handleRemovalOptout(req, res) {
   }
 
   req.session.kanary.onRemovalPilotList = false; //this must be set so they no longer see the tab in the navigation
+  req.session.destroy();
```

Comment:
is line 790 setting var false redundant if we're just destroying the session next line?

Reply:
Hah...good point. I think so. Will remove.